app/models/invenio_rdm_record_converter.rb

```diff
@@ -5,11 +5,14 @@
 #
 class InvenioRdmRecordConverter < Sufia::Export::Converter
   include Galtersufia::GenericFile::InvenioResourceTypeMappings
+  include Galtersufia::GenericFile::KnownOrganizations
 
   # Create an instance of a InvenioRdmRecordConverter converter containing all the metadata for json export
   #
   # @param [GenericFile] generic_file file to be converted for export
-  def initialize(generic_file)
+  def initialize(generic_file=nil)
+    return unless generic_file
```

Comment:
Defaulting the argument to nil was done so that a InvenioRdmRecordConverter instance could be created within the spec without having to supply it a generic file which would call all these functions. With this the spec can just test the creators function. 

I considered moving the creators function somewhere else, but it just doesn't seem to have anywhere else it belongs and especially since we've established a pattern of calling private methods to build out the export json.